core/src/main/java/org/opensearch/sql/executor/ExecutionEngine.java

```diff
@@ -72,9 +80,42 @@ public static class Column {
    * Data class that encapsulates explain result. This can help decouple core engine from concrete
    * explain response format.
    */
-  @Data
   class ExplainResponse {
     private final ExplainResponseNode root;
+    // used in Calcite plan explain
+    private final String logical;
+    private final String physical;
+    private final String codegen;
```

Comment:
I'm thinking shall we fit all these into existing ExplainResponseNode structure?

Reply:
How about add a `ExplainResponseNodeV2`? The new explain response is plain but ExplainResponseNode is nested.